content/html/concepts/tags/terms/abbr/abbr.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '<abbr>'
+Description: 'Indicates an acronym or abbreviation of a longer word or phrase.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'DOM'
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<abbr>` tag indicates an acronym or abbreviation of a longer word or phrase. It can contain the `<title>` attribute which isn't mandatory despite being recommended.
```

Comment:
```suggestion
The `<abbr>` tag indicates an acronym or abbreviation of a longer word or phrase. It can use the `title` attribute which is optional but recommended.
```